lib/feature_management.rb

```diff
@@ -153,4 +153,8 @@ def self.voip_allowed_phones
       Set.new
     end
   end
+
+  def self.ial2_step_indicator_enabled?
+    IdentityConfig.store.ial2_step_indicator_enabled
+  end
```

Comment:
With new type coercions, I wonder if it's worth the effort of having a method in this class, vs. just referencing the store value directly.

Reply:
In cases where the method is just the call to the config, it probably makes sense to do that yeah.

Reply:
> In cases where the method is just the call to the config, it probably makes sense to do that yeah.

Updated in c971983a7b4b11b3d71c62168717e4279087224f.